src/stitch_fix/__tests__/logWeaselTest.test.js

```diff
@@ -1,18 +1,18 @@
 import { LogWeasel } from '../logWeasel';
 
 describe('init', () => {
-  it('needs a key', () => {
-    expect(() => LogWeasel.init()).toThrow();
+  it('throws an error if key is not passed', () => {
+    expect(() => LogWeasel.init()).toThrowErrorMatchingSnapshot();
   });
 });
 
 describe('generateId', () => {
   beforeAll(() => {
-    LogWeasel.init('WEASEL');
+    LogWeasel.init('weasel-gang-service');
```

Comment:
https://www.google.com/search?q=what+is+a+group+of+weasels+called

Reply:
Personally I like "boogle" but "gang" works.

Reply:
I like a "confusion" of weasels. 🤣

Reply:
They're all pretty good 😂